dotnet/src/SemanticKernel.Abstractions/Services/OrderedIAIServiceSelector.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.SemanticKernel.Services;
 internal class OrderedIAIServiceSelector : IAIServiceSelector
 {
     /// <inheritdoc/>
-    public (T?, AIRequestSettings?) SelectAIService<T>(Kernel kernel, SKContext context, ISKFunction skfunction) where T : IAIService
+    public (T?, AIRequestSettings?) SelectAIService<T>(Kernel kernel, SKContext context, KernelFunction skfunction) where T : IAIService
```

Comment:
Can you also search and replace variable names like "skFunction"?

Reply:
Tracking here: https://github.com/microsoft/semantic-kernel/issues/3605